### PR TITLE
Add attribute for responsive

### PR DESCRIPTION
**Pull request by madvik**
Proposition following to #492

### CODE REVIEW

File: templates/subscription.php [hunk 1]
```diff
@@ -24,10 +24,10 @@
 	</thead>
 	<tbody>
 		<tr>
-			<td><?php rcp_print_status(); ?></td>
-			<td><?php echo rcp_get_subscription(); ?></td>
-			<td><?php echo rcp_get_expiration_date(); ?></td>
-			<td>
+			<td data-th="<?php _e( 'Status', 'rcp' ); ?>"><?php rcp_print_status(); ?></td>
```
Comment (mindctrl) [inline templates/subscription.php]: @madvik all these `data-th` should use `esc_attr_e` instead of `_e`

**Comment by mindctrl** [comment]
@madvik you want to work up the required CSS changes?

**Comment by madvik** [comment]
Here the css

**Comment by madvik** [comment]
I change width and column display

File: includes/css/forms.css [hunk 2]
```diff
@@ -133,3 +133,38 @@ p.rcp_success {
 .rcp_current_cards_fieldset span {
     display: block;
 }
+/*
+* RCP-Table for or small screen
+*/
+@media only screen and (max-width: 480px) {
+	.rcp-table{
+	 overflow: hidden;
```
Comment (pippinsplugins) [inline includes/css/forms.css]: The indention on this whole block of CSS needs standardized.

File: includes/gateways/class-rcp-payment-gateway-stripe-checkout.php [hunk 3]
```diff
@@ -48,7 +48,7 @@ public function fields() {
 			'allowRememberMe'   => true,
 			'email'             => $email,
 			'currency'          => rcp_get_currency(),
-			'alipay'            => isset( $rcp_options['stripe_alipay'] ) && '1' === $rcp_options['stripe_alipay'] && 'USD' === rcp_get_currency() ? 'true' : 'false'
+			'alipay'            => isset( $rcp_options['stripe_alipay'] ) && '1' === $rcp_options['stripe_alipay'] && 'USD' === rcp_get_currency() ? true : false
```
Comment (pippinsplugins) [inline includes/gateways/class-rcp-payment-gateway-stripe-checkout.php]: This should be removed from the PR please

**Comment by mindctrl** [comment]
@madvik did you see my comment here? https://github.com/restrictcontentpro/restrict-content-pro/pull/778

File: includes/gateways/class-rcp-payment-gateway-stripe-checkout.php (hunk 3 above)
Comment (mindctrl) [inline includes/gateways/class-rcp-payment-gateway-stripe-checkout.php]: @madvik `git revert d68965a37565f99fed974cc237cca6cc472cc9eb` should do it.

**Comment by madvik** [comment]
Hi, yes I see and I combine, no ?

File: includes/gateways/class-rcp-payment-gateway-stripe-checkout.php (hunk 3 above)
Comment (madvik) [inline includes/gateways/class-rcp-payment-gateway-stripe-checkout.php]: Houch, euh, I don't change alipay configuration...
And I am not comfortable with git, I'm afraid to make mistakes.

**Pull request merged**